src/mya.c

```diff
@@ -394,7 +394,10 @@ int main (int argc, char *argv[]) {
 
 		/* error when anime list is not found due to invalid user */
 		if (!json_object_object_get_ex(json, "data", &anime_list)) {
-			fprintf(stderr, "User not found\n");
+			if (strcmp(CLIENT_ID, "YOUR TOKEN HERE")==0)
```

Comment:
@Sakshi-75, can you add spaces around the `==` here? `HERE") == 0)`

Otherwise, all your changes look good!
Please ignore the Codacy Production warning.

Reply:
@jmakhack done!